src/Directory.Build.props

```diff
@@ -2,9 +2,9 @@
 	<!-- Folder layout -->
 	<PropertyGroup>
 		<IsBenchmarkProject Condition="$(MSBuildProjectName.ToLower().Contains('.benchmarks.'))">true</IsBenchmarkProject>
-		<IsTestProject Condition="$(MSBuildProjectName.ToLower().Contains('.tests.'))">true</IsTestProject>
 		<IsTestAssetProject Condition="$(MSBuildProjectName.ToLower().Contains('tests.shared.'))">true</IsTestAssetProject>
```

Comment:
Another option would be to prefix all your properties in Directory.Build.props by `_`. They seem to be used only here.